src/DataMapper/DataMapperInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusFacebookPlugin\DataMapper;
+
+use Setono\SyliusFacebookPlugin\ServerSide\ServerSideEventInterface;
+
+interface DataMapperInterface
+{
+    /**
+     * Returns true if this data mapper supports the given $source and $target
+     *
+     * @param object $source
```

Comment:
Why isn't the `$source` type hinted with `object` instead of having it in phpdoc?